### ISSUE TITLE
Can't let me delete people even if he doesn't have any invoice or belonging to a company

### ISSUE THREAD

**Issue by Houssemdaoud** [idurar/idurar-erp-crm#1013]
Hello,
I created a person in the people section and gave him an invoice then deleted the invoice and when trying to delete the people it doesn't let me.

**To Reproduce**
1. Go to 'People'
2. Click on 'Add new person'
3. make sure he doesn't belong to any company
4. create an invoice in his name
5. delete the invoice
6. try to delete the person

**Expected behavior**
the created person can't get deleted

**Screenshots**
![Screenshot 2024-03-20 151050](https://github.com/idurar/idurar-erp-crm/assets/146457306/72cdd42a-fabc-4aa0-a652-f381ceb69c59)
![Screenshot 2024-03-20 151152](https://github.com/idurar/idurar-erp-crm/assets/146457306/cfaaa8d7-2494-4560-a0d6-601ee098af18)

**Desktop:**
 - OS: [Windows]
 - Browser [Opera GX]

**Comment by slimskhab** [comment]
Hello,
I tried reproducing it and encountered the same error. I'll investigate further to find the cause.

**Comment by inesmrad** [comment]
hi houssem, 
i reproduced the steps and got the same error, i'll try to fix that.

**Comment by sanskaraggarwal2025** [comment]
Yeah, that's because he becomes client when we assign invoice to the person.
So first we have to delete the client first

**Comment by sanskaraggarwal2025** [comment]
Yeah, first we have to delete that person from client

**Comment by inesmrad** [comment]
> A person can be deleted when he has no invoice and he is not client
> 
> or should it be deleted only when he has no invoice? @inesmrad @Houssemdaoud

i guess even as 'people ' and having no invoice, technically we should be able to delete it.

**Comment by sanskaraggarwal2025** [comment]
@salahlalami Could you please assign this issue to me? I would like to work on it.

**Comment by samr874** [comment]
@Houssemdaoud  tell me what is the status of this issue please tell me??? does it has to be worked upon ??

**Comment by Houssemdaoud** [comment]
> @Houssemdaoud tell me what is the status of this issue please tell me??? does it has to be worked upon ??

If u create a new person then create an invoice in his name then delete the invoice it doesn't let you to delete the person because it says there is an invoice in his name
